hy/__init__.py

```diff
@@ -19,7 +19,11 @@
 # DEALINGS IN THE SOFTWARE.
 
 
-from hy.version import __version__, __appname__  # NOQA
+__appname__ = 'hy'
+try:
+    from hy.version import __version__
+except ImportError:
+    __version__ = 'unknown'
```

Comment:
This whole "unknown version" thing kind of icks me a bit. Here's a thought instead:

- `version.py` contains something like this:

    ```python
    __version__ = '0.12.0' # Or equivalent!
    try:
       from hy._version import git_hash as _git_hash
    except ImportError:
        pass
    else:
        __version__ += 'g' + _git_hash
    ```

And then `setup.py` just writes `_version.py`. That should get rid of the whole issue with `version.py` being non-existent.